src/main/kotlin/org/jitsi/nlj/rtp/codec/vp8/Vp8Packet.kt

```diff
@@ -24,10 +24,36 @@ import org.jitsi.rtp.extensions.bytearray.hashCodeOfSegment
 import org.jitsi_modified.impl.neomedia.codec.video.vp8.DePacketizer
 
 class Vp8Packet(
-    data: ByteArray,
+    buffer: ByteArray,
     offset: Int,
     length: Int
-) : VideoRtpPacket(data, offset, length) {
+) : VideoRtpPacket(buffer, offset, length) {
```

Comment:
one bummer about this is that we have to re-parse the data in each clone, rather than setting it specifically on the clone.  right now i think we happen to read these before the clone (as we'll check these in `wants`, and the result of `wants` is how we decide when to clone), but still it seems a shame to lose the data on clone.

Reply:
What would be nice would be some sort of "clonable lazy" delegate where you could pre-populate the delegate in the clone if the original has already computed it.  There isn't anything like that for Kotlin, though.

Really, all these lazy evals are sort of overkill -- none of these are terribly expensive operations.  I'm mostly just doing this to get the values to show up in the debugger -- lazy delegates do, whereas generic getters don't.